store/index-refactor.js

```diff
@@ -58,5 +70,12 @@ export const mutations = {
             p_state.columnToCategoryMapping[columnName] = targetCategory;
         }
 
+    },
+
+    createColumnToCategoryMap(p_state) {
```

Comment:
based on #259 this guy is supposed to set all mappings to `null`, probably only when the user loads a new `dataTable`. I think `resetColumnToCategoryMap` or something like that (`initialize`?) would probably be more explicit about the intention of this thing. The current name makes me think that the functionality could be very similar to `alterColumnCategoryMapping` which it is not.

Reply:
I'm going to go with 'initialize' as 'reset' doesn't quite make sense on its first (and predominant) usage.